converter/grpc_interceptor_test.go

```diff
@@ -62,118 +58,6 @@ func payloadEncoding(payloads *commonpb.Payloads) string {
 	return string(payloads.Payloads[0].Metadata[MetadataEncoding])
 }
 
-func TestServiceInterceptorRequests(t *testing.T) {
```

Comment:
Can we have some test to confirm that the failure conversion interceptor does as expected?

Reply:
Yep sorry I missed that. Added now.